frontend/src/metabase/public/components/widgets/SharingPane.tsx

```diff
@@ -57,6 +57,7 @@ export default function SharingPane({
   const publicLink = getPublicUrl(resource, extensionState);
   const iframeSource = getPublicEmbedHTML(getPublicUrl(resource));
 
+  const canEmbed = resourceType !== "page";
```

Comment:
why can't we embed pages?

Reply:
I think we actually could, but I'd limit the scope to shared pages for now